docs/src/main/asciidoc/context-propagation.adoc

```diff
@@ -156,6 +156,121 @@ in order to get context propagated.
 The injected `ManagedExecutor` uses the Quarkus thread pool.
 ====
 
+== Overriding which contexts are propagated
+
+By default, all available contexts are propagated. However, you can override this behaviour in several ways.
+
+=== Using configuration
+
+The following configuration properties allow you to specify the default sets of propagated contexts:
+
+[cols="1,1,1"]
+|===
+|Configuration Key|Description|Default Value
+
+|`mp.context.ThreadContext.propagated`
+|The comma-separated set of propagated contexts
+|`Remaining` (all non-explicitly list contexts)
+
+|`mp.context.ThreadContext.cleared`
+|The comma-separated set of cleared contexts
+|`None` (no context), unless neither the propagated nor cleared sets contain `Remaining`, in which case the default is `Remaining` (all non-explicitly listed contexts)
+
+|`mp.context.ThreadContext.unchanged`
+|The comma-separated set of unchanged contexts
+|`None` (no context)
+|=== 
+
+The following contexts are available in Quarkus either out of the box, or depending on whether you include
+their extensions:
+
+[cols="1,1,1"]
```

Comment:
I see no mention of the `Security` context here (see `ThreadContext#SECURITY`) or the `OpenTracing` context (see SmallRye Fault Tolerance's `TracingContextProvider#getThreadContextType()`). Were these omitted by accident or on purpose? Maybe there should be a note explaining this.